pincer/middleware/stage_instance_update.py

```diff
@@ -8,8 +8,8 @@
 from typing import TYPE_CHECKING
 
 from ..objects import StageInstance
-from ..utils.conversion import construct_client_dict
 from ..utils import Coro, replace
+from ..utils.conversion import construct_client_dict
```

Comment:
Can this be combined?
```python
from ..utils import Coro, replace, import construct_client_dict
```

Reply:
maybe ye, but i dont wanna risk ciruclar import, need to be tested first

Reply:
test it :popcorn: